concourse/scripts/build_ip4r.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+CWDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TOP_DIR=${CWDIR}/../../../
+
+source "${TOP_DIR}/gpdb_src/concourse/scripts/common.bash"
+function pkg() {
+    [ -f /opt/gcc_env.sh ] && source /opt/gcc_env.sh
+    source /usr/local/greenplum-db-devel/greenplum_path.sh
+
+    if [ "${OS_NAME}" = "rhel6" ]; then
```

Comment:
https://github.com/greenplum-db/diskquota/pull/225

This have problems, see above link to fix it.

Reply:
In fact, I don't need to set `CC`. So I remove it.